src/app/components/onboarding/profile/profile.component.html

```diff
@@ -0,0 +1,21 @@
+<h2>{{'PROFILE' | translate}}</h2>
+<div><a href="#">{{'navigation.BACKTODASHBOARD' | translate}}</a></div>
```

Comment:
Ich finde das Code schöner wenn man immer ein Leerzeichen am Angang und Ende der {{ ... }} hat - also {{ 'navigation.BACKTODASHBOARD' | translate }}. In dieser pull request wurde es ein bisschen random gemacht

Reply:
ohja.. das hast du Recht, ich eigentlich auch! Vieeel besser zu lesen :-)